src/worker.h

```diff
@@ -34,6 +37,7 @@ class WorkerServiceImpl final : public WorkerService::Service {
   Status ExecuteTask(ServerContext* context, const ExecuteTaskRequest* request, ExecuteTaskReply* reply) override;
   Status ImportFunction(ServerContext* context, const ImportFunctionRequest* request, ImportFunctionReply* reply) override;
   Status Die(ServerContext* context, const DieRequest* request, DieReply* reply) override;
+  Status ImportReusableVariable(ServerContext* context, const ImportReusableVariableRequest* request, AckReply* reply) override;
```

Comment:
So the difference I see is that with the function, we're *actually importing* the function (the code and the whole closure). Whereas with the variable, we're just importing a constructor of the thing, not the thing itself. That's why I didn't feel they're doing the same kind of thing and should be named different. But it's not terribly important, if you don't feel the same way then just keep it.